src/client/opamArg.ml

```diff
@@ -1020,6 +1021,15 @@ let config =
     "Without argument, prints a documented list of all available variables. With \
      $(i,PACKAGE), lists all the variables available for these packages. Use \
      $(i,-) to include global configuration variables for this switch.";
+    ["set"]     , `set     , ["VAR";"VALUE"],
```

Comment:
This would later be expanded for package variables right ? (If #2247 gets implemented).